test/example-types/metricGroups/pivot.js

```diff
@@ -1180,6 +1180,7 @@ describe('pivot', () => {
             'pivotMetric-avg-PO.IssuedAmount': {
               avg: { field: 'PO.IssuedAmount' },
             },
+            metric: { avg: { field: 'PO.IssuedAmount' } },
```

Comment:
This is unfortunate, but fixing this would mean that we need to remove dots from all agg names which would be a much larger change. We can evaluate that later - but doing so would also clean up some of the process response mess.